### DISCUSSION TITLE
Where is the autoload.inc.php file?

### DISCUSSION THREAD

**Discussion by betoxx** [dompdf/dompdf#2679] [Q&A] [answered]
I have downloaded the files but can`t find out autoload.inc.php 
Thank you!

**Comment by betoxx** [comment, marked as answer]
I will answer myself, I have downloaded the zip from the repository that doesn`t include it.

**Reply by bsweeney** [reply]
The source code archive does not include it, but the packaged release should. I generally would not recommend downloading the source code archive since it does not include any dependencies. But if you want to go that route you can get the current autoloader from the utils project: https://github.com/dompdf/utils
